core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -18,18 +18,27 @@
 import alluxio.exception.ExceptionMessage;
 
 import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 /**
  * Tests enum type {@link PropertyKey}.
  */
 public final class PropertyKeyTest {
+  static PropertyKey sDebugAlias;
```

Comment:
- let's not mess up with `PropertyKey.DEBUG`, instead, we can simply create a test property.
- why static member variable? can we simply have the following definition and initialization?
```java
private PropertyKey mTestProperty;

@Before
public void before() {
  mTestProperty = PropertyKey.create("alluxio.test.property", false, new String[] {"alluxio.test.alias1", "alluxio.test.alias2"});
}
```